jdaviz/configs/cubeviz/plugins/slice/tests/test_slice.py

```diff
@@ -1,59 +1,46 @@
 import warnings
 
 import pytest
-import numpy as np
 
 from jdaviz.configs.cubeviz.plugins.slice.slice import Slice
 
 
 def test_slice(cubeviz_helper, spectrum1d_cube):
     app = cubeviz_helper.app
     sl = Slice(app=app)
+#    sl = cubeviz_helper.plugins['Slice']._obj
```

Comment:
Shouldn't we prefer `cubeviz_helper.plugins['Slice']` since that is the API we are advertising?

```suggestion
    sl = cubeviz_helper.plugins['Slice']._obj
```

Reply:
One is instantiated at app startup and the other when called. So they run through different logic routes, see my reply above. We really should test both for all cases, but that would take some work.  Here I ultimately don't care, I tested both by toggling the comment and it works.

Reply:
But CI cannot do manual toggle like you did. If you want to test them both, then please use `pytest.mark.parametrize`. Thanks!

Reply:
ok, I'll just remove the comment then and make a tech debt ticket to consider parameterizing over this in _all_ tests.  I just had this for convenience to test myself.